filesystems/samba2/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+name: samba2
+services:
+  test-samba-share:
+    volumes:
+      - ./smb.conf:/etc/samba/smb.conf
+    ports:
+      - 139:139
+      - 137:137/udp
+      - 138:138/udp
+      - 445:445
+    image: scitotec/test-samba-share
```

Comment:
If we are going to build the image, we should use a different name. e.g. `private.docker.nexus.frankframework.org/ff-test/filesystems/samba2`